### DISCUSSION TITLE
1.5 no slide show?

### DISCUSSION THREAD

**Discussion by jdonwells** [TEALSK12/introduction-to-computer-science#322] [Show and tell]
1.5 doesn't seem to have a slide show to go with it. So I made one if you want to use it.[https://docs.google.com/presentation/d/1yz2Ef_XU_GvWFoGvmkPoOOKGjqtvvugCpk6BmSKVtLY/edit#slide=id.p1](https://docs.google.com/presentation/d/1yz2Ef_XU_GvWFoGvmkPoOOKGjqtvvugCpk6BmSKVtLY/edit#slide=id.p1)

**Comment by jpfasano** [comment]
I tried to access the google doc link, but didn't have the required access permissions.

**Reply by jdonwells** [reply]
I changed the permissions to anyone with the link. https://docs.google.com/presentation/d/1yz2Ef_XU_GvWFoGvmkPoOOKGjqtvvugCpk6BmSKVtLY/edit?usp=sharing

**Reply by jpfasano** [reply]
Thank you.
